taskcluster/scripts/check-screenshots.py

```diff
@@ -36,37 +38,56 @@ def _init_logging():
     )
 
 
+def _parse_screenshots_configuration(config_file):
+    config = yaml.safe_load(config_file)
+
+    number_of_screenshots_per_locale = {}
+    for locale in config["locales"]:
```

Comment:
This loop is in charge of counting how many screenshots a locale has. As of now a locale can have either 55 or 54 pictures. We may have other kinds of locales that bump/reduce this number. Each new kind will have to be defined here.